mypy/build.py

```diff
@@ -828,11 +828,11 @@ def _load_json_file(file: str, manager: BuildManager,
     try:
         result = json.loads(data)
     except ValueError:  # TODO: JSONDecodeError in 3.5
-        manager.errors.report(0, 0, "Error reading JSON file {file};"
+        manager.errors.set_file(file, None)
+        manager.errors.report(0, 0, "Error reading JSON file;"
```

Comment:
I'd use `-1, -1` for the line/col, and I prefer the long string to start on the next line, so as not to hide the first two args in front of it.

Otherwise LG!